src/clr-core/common/utils/aria-live.service.spec.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { AriaLivePoliteness } from '@clr/core/common';
```

Comment:
Is this staying in?

Reply:
Nope deleting soon but since my PR's Angular version now extends from it, didn't want to  delete yet because it'll break all the Angular components that extend it. Will be deleted when I revert all the aria-live changes in this PR.